### PR TITLE
Copy to clipboard example

### PR DESCRIPTION
**Pull request by bahmutov**
Uses the custom element that GitHub uses to copy every code fragment. Currently works pretty well in Electron only, other browsers probably will need a native event or some permission stuff. But it is a good start.

copy-paste

The test failures are from flakey tests in unrelated recipes

### CODE REVIEW

**Comment by bahmutov** [comment]
good example for https://github.com/cypress-io/cypress/issues/2752

File: examples/testing-dom__clipboard/cypress/integration/permissions-spec.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  it('can be queried in Chrome', { browser: 'chrome' }, () => {
+    cy.visit('index.html') // yields the window object
+    .its('navigator.permissions')
+    // permission names taken from
+    // https://w3c.github.io/permissions/#enumdef-permissionname
+    .invoke('query', { name: 'clipboard-read' })
+    .its('state').should('be.oneOf', ['prompt', 'granted', 'denied'])
+  })
+
+  it('can be granted in Chrome', { browser: 'chrome' }, () => {
```
Comment (jennifer-shehane) [inline examples/testing-dom__clipboard/cypress/integration/permissions-spec.js:28]: This example is confusing, it makes it seem like you can test clipboard in Chrome by overwriting the permissions, but you say it's impossible due to the popup. I would remove this. What's the use of this test practically?

File: examples/testing-dom__clipboard/cypress/integration/spec.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    cy.window().its('navigator.clipboard')
+    .invoke('readText')
+    .should('equal', 'npm install -D cypress')
+    .then((text) => {
+      // paste the text from the clipboard into the text area
+      cy.get('#paste-here').click().invoke('val', text)
+    })
+
+    // take a screenshot to show the pasted text
+    cy.screenshot()
```
Comment (jennifer-shehane) [inline examples/testing-dom__clipboard/cypress/integration/spec.js:47]: I would remove these screenshots. It makes the test visually jump around for each screenshot. I was confused. It also makes it seem that screenshotting is necessary to see the pasted text, but I can see it in the UI.

**Review by jennifer-shehane — CHANGES_REQUESTED** [review]
Why does this functionally only work in Electron? Is Cypress sending a flag for Electron to make it skip clipboard permissions that you know of?

File: examples/testing-dom__clipboard/cypress/integration/spec.js (hunk 2 above)
Comment (bahmutov) [reply examples/testing-dom__clipboard/cypress/integration/spec.js:47]: Ok, I will remove it, wanted to show the screenshots on CI, but not necessary if confusing

File: examples/testing-dom__clipboard/cypress/integration/permissions-spec.js (hunk 1 above)
Comment (bahmutov) [reply examples/testing-dom__clipboard/cypress/integration/permissions-spec.js:28]: Yup, in Chrome we can grant the permissions using automation CDP call

**Pull request merged**